samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -0,0 +1,162 @@
+.. _direction_finding_connectionless_rx:
+
+Bluetooth: Direction finding connectionless locator
+###################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833
+
+The sample also requires an antenna matrix when operating in Angle of Arrival mode.
```

Comment:
I don't see any changes for AoA and AoD (regarding moving to lower case). Is that intentional?

Reply:
capitals changed to lowercase. Sorry, I've forgotten about that.